recipes/emperor/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0beta4" %}
```

Comment:
So we do want to figure out a way to do pre-releases. However we are still discussing how. See this issue ( https://github.com/conda-forge/conda-forge.github.io/issues/147 ). We would welcome your feedback to the discussion. Would it be alright to do an official release to get this out there?